sklearn/datasets/base.py

```diff
@@ -255,14 +255,25 @@ def load_iris():
 
     Read more in the :ref:`User Guide <datasets>`.
 
+    Parameters
+     ----------
+    return_X_y : boolean, default=False.
+        If True, returns (data, target) instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
     Returns
     -------
+    if return_X_y == false
     data : Bunch
         Dictionary-like object, the interesting attributes are:
         'data', the data to learn, 'target', the classification labels,
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    
+    if return_X_y == true
+    (data,target) : tuple
```

Comment:
I think this would look better in the docs as  `(data, target) : tuple if return_X_y is True`...I can't think of any precedent for this in the code base; could you generate the docs by running `make html-noplot` in the `docs/` directory and post a screenshot of what it looks like?